internal/host/transition_test.go

```diff
@@ -834,15 +823,7 @@ var _ = Describe("Install", func() {
 			It(t.name, func() {
 				host = hostutil.GenerateTestHost(hostId, infraEnvId, clusterId, t.srcState)
 				Expect(db.Create(&host).Error).ShouldNot(HaveOccurred())
-				mockEvents.EXPECT().SendHostEvent(gomock.Any(), eventstest.NewEventMatcher(
```

Comment:
Why was this assertion removed?

Reply:
This expectation was removed because all these transitions result in errors like this:

```
no condition passed to run transition InstallHost from state preparing-successful
```

When that happens the current code doesn't generate any event, so the expectation is never called.